packages/kit/src/runtime/server/endpoint.js

```diff
@@ -33,7 +33,7 @@ export default async function render_route(request, route) {
 				);
 			}
 
-			let { status = 200, body, headers = {} } = response;
+			let { status = 200, body = {}, headers = {} } = response;
```

Comment:
I think we should only do this for JSON `content-type` and not for text.

Might be able to do that by leaving this as it was and tweaking lines 58 & 62 or something

Reply:
That might be better, I was also considering the same recently